### ISSUE TITLE
Improve Behavior when No Virtual Robots.txt

### ISSUE THREAD

**Issue by bobmatyas** [bobmatyas/wp-block-ai-crawlers#34]
Rather than just erroring out when a `robots.txt` file is present, the plugin should provide the suggest values to add and instructions on how to do so.

**Comment by 2br-2b** [comment]
I think that defeats the purpose of the plugin. The point of the plugin is to automatically add these to the robots.txt file. If you're manually editing robots.txt, you don't need the plugin.

**Comment by bobmatyas** [comment]
I agree, but I'd like to do something in the future to offer a more graceful error state. 

It's a low priority though
